staging/prometheus-operator/values.yaml

```diff
@@ -448,7 +448,7 @@ grafana:
     hosts: []
 
     ## Path for grafana ingress
-    path: /
+    path: /ops/portal/grafana
```

Comment:
Could I get more info why this change was made and why here? 

This does not seem like the right place to do this as it is the values file. Did you mean to make this change in the Kubernetes-base-addons repo?

Reply:
This change addressed a bug where grafana would create incorrect urls to other dashboards within itself: https://jira.mesosphere.com/browse/DCOS-53598. I made the commit in this repo because I didn’t think about the kubernetes-base-addons repo. In retrospect I could see it belonging there instead.

Reply:
The less changes to the forked prometheus-operator chart the better. Overriding in base addons is better IMO.

Reply:
+1 on fewer changes to the forked chart. Also, this ingress is actually disabled so it doesn't look like we're actually adding it. Doesn't look like we set is in [kubernetes-base-addons](https://github.com/mesosphere/kubernetes-base-addons/blob/master/addons/prometheus/0.34.x/prometheus-1.yaml#L251) either, so just checking if this would actually fix the bug? Looks like the ingress is actually configured in the [`opsportal` chart](https://github.com/mesosphere/charts/blob/dev/stable/opsportal/values.yaml#L31)

Reply:
So should we revert this PR since it was merged a few hours ago... @br-lewis thoughts?